disbatch/disBatch.py

```diff
@@ -0,0 +1,1533 @@
+#!/usr/bin/env python3
```

Comment:
This is going to take a while to review without diffs. I'll check it out when I can